React/Modules/RCTAsyncLocalStorage.m

```diff
@@ -67,7 +67,11 @@ static void RCTAppendError(NSDictionary *error, NSMutableArray<NSDictionary *> *
   static NSString *storageDirectory = nil;
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
+#if TARGET_OS_TV
+    storageDirectory = NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES).firstObject;
```

Comment:
AsyncLocalStorage has some guarantees on the persistency of the data. NSCachesDirectory can be removed by the OS at any point. Is there no equivalent directory on tvOS?